### ISSUE TITLE
code release for edicho?

### ISSUE THREAD

**Issue by Njasa2k** [EzioBy/edicho#1]

**Comment by ApolloRay** [comment]
I hope so.

**Comment by EzioBy** [comment]
Yes, we’re preparing to release the code. Please stay tuned for more information.

**Comment by KleinXin** [comment]
soon

**Comment by venuv** [comment]
any updates on the code release time frame?

**Comment by animemory** [comment]
> Yes, we’re preparing to release the code. Please stay tuned for more information.

hi, how is it going now?

**Comment by EzioBy** [comment]
Thanks for your attention. Codes will be released once the paper is accepted.

**Comment by Tonniia** [comment]
hi, how is it going now?

**Comment by KleinXin** [comment]
> Thanks for your attention. Codes will be released once the paper is accepted.

Has it already been accepted by ICCV2025？

**Comment by EzioBy** [comment]
Yes. Thanks for your patience. We plan to release the code within this month.
